client/src/lib/attempt-result.js

```diff
@@ -256,18 +256,71 @@ export function autocompleteTimeAttemptResult(time) {
   return roundOver10Mins(time);
 }
 
+/**
+ * Checks if a given set of results is a new world record average and returns a corresponding boolean.
+ */
+export function isWorldRecordAverage(
+  attemptResults,
+  eventId,
+  officialWorldRecords
+) {
+  const newAverage = average(attemptResults, eventId);
+  const wrAverage = officialWorldRecords?.find(
+    (wr) => wr.type === 'average' && wr.event.id === eventId
+  )?.attemptResult;
+  // if no wrAverage is found for some reason (networking issue, brand new event, etc.),
+  // we get `number < undefined` which always evaluates to `false`
```

Comment:
We should always have the records, they are fetched together with the round, so if there are networking issues then we wouldn't load the page in the first place :)

Reply:
With that in mind, I removed the comment altogether